policyengine_uk/tests/microsimulation/test_childcare.py

```diff
@@ -69,6 +108,45 @@ def test_extended_childcare_entitlement_aggregate():
     )
 
 
+def test_extended_childcare_entitlement_recipients():
```

Comment:
Something like this should definitely be wrapped in a function we call, we want to avoid code duplication